client/src/modules/cohort/components/CohortHeader.jsx

```diff
@@ -23,10 +23,10 @@ class CohortHeader extends PureComponent {
     const trimmedDescription = description.trim();
 
     this.state = {
-      description: trimmedDescription,
+      descriptionInput: trimmedDescription,
       isDescriptionTextareaVisible: false,
       isNameInputVisible: false,
-      name,
+      nameInput: name,
```

Comment:
check previous comment (nameValue) or even nameInputValue and descriptionInputValue